### PR TITLE
Update cortana-integration.md

### PR DESCRIPTION
**Pull request by msbemba**
Added method to find out usage location - Reference https://docs.microsoft.com/en-us/microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell?view=o365-worldwide#view-additional-property-values-for-accounts

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2594

File: microsoft-365/admin/misc/cortana-integration.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Moving forward, we're focusing Cortana on enterprise productivity.
...
 
-- New Microsoft 365 experiences, such as the Briefing email and Play My Emails, will be enabled using Cortana enterprise services and fully comply with those promises. These features are currently available in the English (United States) market for users with mailboxes in the US geo, and for employees of multi-national corporations with US-based mailboxes.
+- New Microsoft 365 experiences, such as the Briefing email and Play My Emails, will be enabled using Cortana enterprise services and fully comply with those promises. These features are currently available in the English (United States) market for users with mailboxes in the US geo, and for employees of multi-national corporations with US-based mailboxes. For more information on finding the usage location, Please visit [View additional property values for accounts](https://docs.microsoft.com/microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell?view=o365-worldwide#view-additional-property-values-for-accounts)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/cortana-integration.md]:
```suggestion
- New Microsoft 365 experiences, such as the Briefing email and Play My Emails, will be enabled using Cortana enterprise services and fully comply with those promises. These features are currently available in the English (United States) market for users with mailboxes in the US geo, and for employees of multi-national corporations with US-based mailboxes. For more information on finding the usage location, please visit [View additional property values for accounts](https://docs.microsoft.com/microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell?view=o365-worldwide#view-additional-property-values-for-accounts).
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/misc/cortana-integration.md]: Please add a period after the link and lowercase the "p" in "Please".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**